README.md

```diff
@@ -21,22 +21,17 @@ npx @eslint/create-config
 If you want to use a specific shareable config that is hosted on npm, you can use the `--config` option and specify the package name:
 
 ```bash
-# use `eslint-config-semistandard` shared config
+# use `eslint-config-standard` shared config
 
 # npm 7+
-npm init @eslint/config -- --config semistandard
-
-# or (`eslint-config` prefix is optional)
-npm init @eslint/config -- --config eslint-config-semistandard
+npm init @eslint/config -- --config eslint-config-standard
 
 # ⚠️ npm 6.x no extra double-dash:
-npm init @eslint/config --config semistandard
+npm init @eslint/config --config eslint-config-standard
```

Comment:
Would this actually work with npm 6? Node.js 18.18.0 comes with npm 9.8.1. I'm not sure if mentioning npm 6 is necessary.